exercises/affine-cipher/description.md

```diff
@@ -1,14 +1,12 @@
 # Description
 
-Create an implementation of the affine cipher,
-an ancient encryption system created in the Middle East.
+Create an implementation of the affine cipher, an ancient encryption system created in the Middle East.
 
-The affine cipher is a type of mono-alphabetic substitution cipher.
-Each character is mapped to its numeric equivalent, encrypted with
-a mathematical function and then converted to the letter relating to
-its new numeric value. Although all mono-alphabetic ciphers are weak,
-the affine cipher is much stronger than the atbash cipher,
-because it has many more keys.
+The affine cipher is a type of monoalphabetic substitution cipher.
+Each character is mapped to its numeric equivalent, encrypted with a mathematical function and then converted to the letter relating to its new numeric value.
+Although all monoalphabetic ciphers are weak, the affine cipher is much stronger than the atbash cipher, because it has many more keys.
+
+[comment]: # ( monoalphabetic as spelled by Merriam-Webster, compare to polyalphabetic )
```

Comment:
I wouldn't have required that this comment be preserved in the file, but I'm fine either way.

Reply:
I added this in the file, because it is not the first time this word has been modified.  Having the comment in the file hopefully gives the reasoning as to why it is the way it is.  (when changing the spelling again, it is more likely this reference will be seen.)